src/scripts/OSFramework/OSUI/Pattern/InlineSvg/IInlineSvg.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSUI.Patterns.InlineSvg {
+	/**
+	 * Defines the interface for OutSystemsUI InlineSvg Pattern
+	 */
+	export interface IInlineSvg extends Interface.IPattern {
+		registerCallback(eventName: string, callback: GlobalCallbacks.OSGeneric): void;
```

Comment:
You don't need this method here,  you can add the the Interface.ICallback, after the IPattern

Reply:
Fixed